src/test/ui/proc-macro/musl-proc-macro.rs

```diff
@@ -0,0 +1,15 @@
+// Test proc-macro crate can be built without addtional RUSTFLAGS
+// on musl target
+
+// build-pass
+// only-musl
```

Comment:
It depend on musl target but CI don't have it.
```log
2020-03-03T09:46:02.3188089Z error[E0463]: can't find crate for `std`
2020-03-03T09:46:02.3188535Z    |
2020-03-03T09:46:02.3189112Z    = note: the `x86_64-unknown-linux-musl` target may not be installed
2020-03-03T09:46:02.3189589Z error: aborting due to previous error
2020-03-03T09:46:02.3189910Z 
2020-03-03T09:46:02.3190426Z For more information about this error, try `rustc --explain E0463`.
```

Reply:
Which CI job was it? This is a problem with the CI configuration, not something that needs to be worked around in the test. Specifically, the musl targets built by CI are missing the dynamic version of `libstd`. To fix this, they need to be built with `crt_static=false` (the `config.toml` option, not the rustc flag), like the `dist-x86_64-musl` job does: https://github.com/rust-lang/rust/blob/1d5241c96208ca7d925442b1a5fa45ad18717a6f/src/ci/docker/dist-x86_64-musl/Dockerfile#L37

Reply:
@smaeul it failed on `x86_64-gnu-llvm-7`.

Reply:
The `x86_64-gnu-llvm-7` job only builds for `x86_64-unknown-linux-gnu`. It doesn't look like it does anything with musl at all. So where did `"--target=x86_64-unknown-linux-musl"` in the rustc arguments for this test come from?

Reply:
Oh, I see, the old version of the test had `compile-flags: --target=x86_64-unknown-linux-musl` in it. That was the problem. After removing that line, the test should work on all targets.

So since the behavior tested here is not musl-specific, I would suggest 1) removing `only-musl` and 2) renaming the test to mention `crt-static` instead of `musl`.